qml/PolarGrid.qml

```diff
@@ -103,4 +103,29 @@ Item {
             }
         }
     }
+
+    Canvas {
+        anchors.fill: parent
+        opacity: 0.8
+        property var angle: 45
+
+        onPaint: {
```

Comment:
Does this redraw every time we paint a new profile? If so, is that necessary? Can it just belong on another layer/item?

Reply:
No. What is necessary ? Why it should be in another item ? This is the polar grid, it should deal with grid stuff.

Reply:
My question is:

Does **this item** get drawn/painted once at initialization, or repeatedly every time the visualizer updates the sensor data?

Reply:
(when is onPaint called)

Reply:
As said before. No.
Paint is only called when the region needs to be rendered, like resize events.